lichess/api.py

```diff
@@ -321,6 +321,11 @@ def tv_channels(**kwargs):
     """Wrapper for the `GET /tv/channels <https://github.com/ornicar/lila#get-tvchannels-fetch-current-tournaments>`_ endpoint."""
     return _api_get('/tv/channels', kwargs)
 
+def cloud_evaluation(**kwargs):
```

Comment:
(1) Let's call this cloud_eval to map directly to the api.
(2) Let's add `fen` as an explicit parameter since it's required (and add it to the kwargs dict before calling apiget).